filesystems/cvmfs/fs.go

```diff
@@ -0,0 +1,58 @@
+// +build linux
+
+package main
+
+import (
+	"fmt"
+	"strings"
+	"path/filepath"
+	"os"
+	"syscall"
+
+	"github.com/containerd/containerd/plugin"
+	fsplugin "github.com/ktock/remote-snapshotter/filesystems"
+)
+
+func init() {
+	plugin.Register(&plugin.Registration{
+		Type:   fsplugin.RemoteFileSystemPlugin,
+		Config: &Config{},
+		ID:     "cvmfs",
+		InitFn: func(ic *plugin.InitContext) (interface{}, error) {
+			config, ok := ic.Config.(*Config)
+			if !ok {
+				return nil, fmt.Errorf("invalid stargz configuration")
+			}
+			ic.Meta.Exports["root"] = ic.Root
+
+			return NewFilesystem(ic.Root, config)
+		},
+	})
+}
+
+type filesystem struct {}
+
+type Config struct {}
+
+func NewFilesystem(root string, config *Config) (*filesystem, error) {
+	return &filesystem{}, nil
+}
+
+func (fs *filesystem) Mount(ref, digest, moutpoint string) error {
+	digest = strings.Split(digest, ":")[1]
+	firstTwo := digest[0:2]
+	path := filepath.Join("/", "cvmfs", "unpacked.cern.ch", ".layer", firstTwo, digest)
```

Comment:
`unpacked.cern.ch` should not be hardcoded

Reply:
There are a lot of stuff quite specific to our setup at the moment.

I don't expect this to be even close to production. It was mostly a prototype for other people to see.

Sorry if the intentions were unclear!